multichain-testing/yarn.lock

```diff
@@ -1938,26 +1931,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"eslint-plugin-prettier@npm:^5.1.3":
```

Comment:
How / when can we make the Multichain E2E Testing job part of `force-integration`?

Reply:
@toliaqat, happen to know how we can add Multichain E2E Testing job to `force-integration`?

Based on [history](https://github.com/Agoric/agoric-sdk/actions/workflows/multichain-e2e.yml), successful runs are all under 17 mins and failed runs take several minutes.

Reply:
Thanks @LuqiPan @toliaqat. I am sending this PR to the merge queue, but interested to see where we land on this.